### ISSUE TITLE
Sub as a rank

### ISSUE THREAD

**Issue by chalenged** [bufutda/bufbot-issues#27]
I feel subscriber should be a rank for the bot.

**Comment by bufutda** [comment]
What would be the perks? another command layer?

**Comment by chalenged** [comment]
yes, there would be commands exclusive to subscribers and higher. And i
believe subscribers should be between regular and voice ranks.

On Tue, May 26, 2015 at 9:10 PM, bufutda notifications@github.com wrote:

> What would be the perks? another command layer?
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/bufutda/bufbot-issues/issues/27#issuecomment-105747762
> .

## 

Life is like a box of chocolates, unless you don't like chocolate. Then
it's more like a carton of ice cream. Make that a gallon...or 2.

**Comment by firynth** [comment]
Recent changes to chat flags should make this viable right?

**Comment by bufutda** [comment]
Yes.
